misc/aws-k8s-cni-calico.yaml

```diff
@@ -151,48 +157,57 @@ spec:
               name: var-run-calico
               readOnly: false
       volumes:
-        # Used by calico/node.
+        # Used to ensure proper kmods are installed.
         - name: lib-modules
           hostPath:
             path: /lib/modules
         - name: var-run-calico
           hostPath:
             path: /var/run/calico
+      tolerations:
+        # Make sure calico/node gets scheduled on all nodes.
+        - effect: NoSchedule
+          operator: Exists
+        - effect: NoExecute
```

Comment:
Should we pick this change up from what has been done to the Calico manifests? Seems reasonable to me but I wanted to bring it up specifically.

Reply:
Yeah I think so